core/src/impl/Kokkos_Profiling.cpp

```diff
@@ -46,16 +46,16 @@
 #include <impl/Kokkos_Profiling.hpp>
 #if defined(KOKKOS_ENABLE_PROFILING)
 #include <dlfcn.h>
-#endif
-#include <vector>
-#include <array>
-#include <stack>
-
-#if defined(KOKKOS_ENABLE_PROFILING)
 #include <cstring>
 #include <unordered_map>
 #include <unordered_set>
 #include <algorithm>
+#include <vector>
+#include <array>
+#include <stack>
+#include <chrono>
```

Comment:
Where is this used?

Reply:
No idea; just trying to minimize the change set, but since I was changing headers and stuff I went ahead and combined two `#ifdef`s that should have been. We can revert these changes if you want.

Reply:
I've seen your answer and I still don't understand why `<chrono>` was added.